test/parallel/test-process-resource-usage.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const usage = process.resourceUsage();
+
+assert(usage.userCpuTimeUsedSec >= 0);
```

Comment:
Would it be better if we tighten the tests by checking if the values are there for sure in Windows atleast?